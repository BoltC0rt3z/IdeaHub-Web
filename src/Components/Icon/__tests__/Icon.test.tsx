import React from 'react';
import { mount, ReactWrapper } from 'enzyme';
import { faCalendar } from '@fortawesome/free-regular-svg-icons';

import { Icon } from '..';

describe('Tests for Icon', () => {
  const wrapper: ReactWrapper = mount(<Icon icon={faCalendar} />);

  it('Renders icon without crushing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});