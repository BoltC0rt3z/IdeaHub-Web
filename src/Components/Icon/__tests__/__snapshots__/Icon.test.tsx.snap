// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Icon Renders icon without crushing 1`] = `ReactWrapper {}`;
